model_config_file:
  src/legged_base/config/g1_29dof-config.yaml

log_path:
  plot/data/

ll_dt: 0.002
use_sim_timer: false

kp: [
  100,100,100,150,40,40,
  100,100,100,150,40,40,
  200,200,200,
  40,40,40,40,40,40,40,
  40,40,40,40,40,40,40,
]
kd: [
  2,2,2,4,2,2,
  2,2,2,4,2,2,
  5,5,5,
  10,10,10,10,10,10,10,
  10,10,10,10,10,10,10,
]

FixStand:
  q: [ 
    -0.1, 0,0,0.3,-0.2,0,
    -0.1, 0,0,0.3,-0.2,0,
    0,0,0,
    0.35, 0.18,0,0.87,0,0,0,
    0.35,-0.18,0,0.87,0,0,0
  ]

PreIDLE:
